functions/npm.fish

```diff
@@ -1,3 +1,3 @@
-function npm -d "javascript package manager"
+function npm -d "javascript package manager" -c npm -w npm
```

Comment:
-c is a flag for `complete`, not `function`. I haven't tested this, as I don't have that problem with verman, but I am skeptical this is even working haha.